### ISSUE TITLE
Manager Cannot Review Submission for Admin Set

### ISSUE THREAD

**Issue by shlake** [samvera-labs/nurax-pre2023#119]
### Descriptive summary
Account "sherrylake@comcast.net" has been assigned a manager of the "UVa Admin Set". Work flow is set to One-step Mediated. The pending submission does  not show up on the Manager's "Review Submission" list.

### Rationale

Managers are suppose to be able to approve of mediated workflows. But the specific work to approve is not showing up.

### Expected behavior
Should see a list of submitted works, with one-set mediated workflow. Here is the Admin set where "sherrylake@comcast.net" is a manager (oh, it would really be helpful if the Admin Set name was displayed on this tab, take my word for it this is the "UVa Admin Set" list of Managers):
![screen shot 2017-10-23 at 1 50 02 pm](https://user-images.githubusercontent.com/3809325/31904889-6dc13fdc-b7fa-11e7-96eb-2508b75e6d2a.png)

### Actual behavior

So as manager, sherrylake should see works pending, but instead sees this:
![screen shot 2017-10-23 at 1 52 44 pm](https://user-images.githubusercontent.com/3809325/31904980-b3e220ee-b7fa-11e7-9a4c-253d1c9b34d7.png)

The pending work is "pending" as seen here on the Administrators dashboard:
![screen shot 2017-10-23 at 2 06 00 pm](https://user-images.githubusercontent.com/3809325/31905206-646eb904-b7fb-11e7-9ab4-28c2f3c9efdc.png)

### Steps to reproduce the behavior

1. On Admin Set, set workflow "one step"
1. set Participants - make one a "Manager". 
1. 3rd user (not admin nor the "manager") create a work in the admin set
1. On Manager dashboard, click "Review Submissions

**Comment by julesies** [comment]
@shlake did you add the work before or after sherrylake became a manager? the manager permissions do not work as you would expect (which is a known shortcoming of permissions capabilities). Basically managers can only see works that have been added after they became a manager. If you added the manager first, then the work, that is a bug. Let me know and I'll confirm.

**Comment by shlake** [comment]
@julesies Actually, what you said above is partly the problem. I added a new work, title "Verifying....", (after sherrylake was a manager), and yes, I see that pending work. BUT now I also see the 1st work (which I think was submitted when sherrylake was not a manager, and that I did not see when I created this ticket), title: "UVa Admin set is Public":
![screen shot 2017-10-23 at 2 15 50 pm](https://user-images.githubusercontent.com/3809325/31905706-0e3a9718-b7fd-11e7-8a29-b3229d9d1ce3.png)

**Comment by julesies** [comment]
Ok well that is odd! Thanks @shlake can you see, edit the record for UVa Admin set is Public now?

**Comment by julesies** [comment]
Also @shlake tell me if you can edit/view both records.

**Comment by shlake** [comment]
As the Manager, I can view both records. I can click the "Review and Approval" and see the options for both works. I didn't follow through, should I one of them?

**Comment by julesies** [comment]
can you edit the metadata?  I'm more curious about your permissions on the first one that you really shouldn't see...(although it's more what you think would happen)

**Comment by shlake** [comment]
yes, I changed the title on the 1st one (one that did not show up originally).

**Comment by vantuyls** [comment]
need to dig in and test this more thoroughly.
